### DISCUSSION TITLE
Build for Raspberry Pi (3 & 4) fails

### DISCUSSION THREAD

**Discussion by brunokozina** [jens-maus/thinRoot#20] [Q&A] [answered]
Hi,
I try to build for Raspberry Pi 4 (rpi4), but it fails with error:
Makefile.legacy:9: *** "You have legacy configuration in your .config! Please check your configuration.".  Stop.
It does also for Raspberry Pi 3 (rpi3). I tried it on different systems (SLES12, 15, RHEL 7, 8).
For which make version this works (if it is version related)?
Regards,
Bruno

**Comment by jens-maus** [comment, marked as answer]
Sorry for being so late regarding your discussion thread on rpi3/rpi4 support. Please note, that now building for RaspberryPi3 as well as RaspberryPi4 should work again and should be supported. In addition, the RaspberryPi5 should also be supported now. So please enjoy and let me know if everything builds nicely now!
